tests/conftest.py

```diff
@@ -221,3 +221,14 @@ def sanic_ext(ext_instance):  # noqa
     yield sanic_ext
     with suppress(KeyError):
         del sys.modules["sanic_ext"]
+
+
+@pytest.fixture
+def urlopen():
+    urlopen = Mock()
+    urlopen.return_value = urlopen
+    urlopen.__enter__ = Mock(return_value=urlopen)
+    urlopen.__exit__ = Mock()
+    urlopen.read = Mock()
+    with patch("sanic.worker.inspector.urlopen", urlopen):
+        yield urlopen
```

Comment:
Since this will only work for the `sanic.worker` namespace, probably better to move it inside the `tests/worker` folder.

Reply:
`tests_cli.py` uses it too